configure.in

```diff
@@ -80,6 +80,25 @@ FLFEMDEM="femdem"
 # Check system type
 AC_CANONICAL_HOST
 
+case $host_os in
+     darwin* ) 
+	     LINUX=no
```

Comment:
One last indentation issue - this is to proof I actually read the diff again ;-)